backend/src/serverless/integrations/webhooks/github.ts

```diff
@@ -379,6 +379,7 @@ export default class GitHubWebhook {
               location: fromAPI.location ?? null,
               logo: fromAPI.avatarUrl ?? null,
               url: fromAPI.url ?? null,
+              github: { handle: fromAPI.resourcePath },
```

Comment:
shouldn't this be `fromApi.url` ? The graphql response has the gh organization link in `url`

Reply:
At least from sample data the url is different from the handles. Handles should indicate platform profiles/handlers right? For instance, for crowd, the url could be `https://www.crowd.dev/` and the github handle should be `CrowdDotDev`